.drone.star

```diff
@@ -704,39 +746,36 @@ def uiTestPipeline(ctx, filterTags, early_fail, runPart = 1, numberOfParts = 1,
-                    "EXPECTED_FAILURES_FILE": "/drone/src/tests/acceptance/expected-failures-webUI-on-%s-storage%s.md" % (storage.upper(), expectedFailuresFileFilterTags),
-                    "MIDDLEWARE_HOST": "http://middleware:3000",
-                },
-                "commands": [
-                    ". /drone/src/.drone.env",
-                    "git clone -b master --depth=1 https://github.com/owncloud/testing.git /srv/app/testing",
-                    "git clone -b $WEB_BRANCH --single-branch --no-tags https://github.com/owncloud/web.git /srv/app/web",
-                    "cd /srv/app/web",
-                    "git checkout $WEB_COMMITID",
-                    "cd tests/acceptance/",
-                    "yarn install --immutable",
-                    "./run.sh",
-                ],
-            },
-        ] + failEarly(ctx, early_fail),
+                 restoreBuildArtifactCache(ctx, "tests-yarn", "/webTestRunner/tests/acceptance/.yarn") + installWebTestRunner() + yarnInstallUITests() +
+                 ocisServer(storage, accounts_hash_difficulty, [stepVolumeOC10Tests]) + waitForSeleniumService() + waitForMiddlewareService() +
+                 [
+                     {
+                         "name": "webUITests",
+                         "image": OC_CI_NODEJS % DEFAULT_NODEJS_VERSION,
+                         "environment": {
+                             "SERVER_HOST": "https://ocis-server:9200",
+                             "BACKEND_HOST": "https://ocis-server:9200",
+                             "RUN_ON_OCIS": "true",
+                             "OCIS_REVA_DATA_ROOT": "/srv/app/tmp/ocis/owncloud/data",
+                             "TESTING_DATA_DIR": "/srv/app/testing/data",
+                             "WEB_UI_CONFIG": "/drone/src/tests/config/drone/ocis-config.json",
+                             "TEST_TAGS": finalFilterTags,
+                             "LOCAL_UPLOAD_DIR": "/uploads",
+                             "NODE_TLS_REJECT_UNAUTHORIZED": 0,
+                             "RUN_PART": runPart,
+                             "DIVIDE_INTO_NUM_PARTS": numberOfParts,
+                             "EXPECTED_FAILURES_FILE": "/drone/src/tests/acceptance/expected-failures-webUI-on-%s-storage%s.md" % (storage.upper(), expectedFailuresFileFilterTags),
+                             "MIDDLEWARE_HOST": "http://middleware:3000",
+                         },
+                         "commands": [
+                             "git clone -b master --depth=1 https://github.com/owncloud/testing.git /srv/app/testing",
+                             "cd /webTestRunner/tests/acceptance",
+                             "./run.sh",
+                         ],
+                     },
+                 ] + failEarly(ctx, early_fail),
         "services": selenium() + middlewareService(),
-        "volumes": [pipelineVolumeOC10Tests] +
+        "volumes": [pipelineVolumeOC10Tests, stepVolumeOC10Tests] +
```

Comment:
pipeline and step volumes have different usage. Also, this change might not be needed.
```suggestion
        "volumes": [pipelineVolumeOC10Tests] +
```

Reply:
that got left out. Removed now! Thanks for pointing it out!